scalameta/parsersJSFacade/src/main/scala/scala/meta/parsersjsfacade/Api.scala

```diff
@@ -0,0 +1,59 @@
+package scala.meta
+package parsersjsfacade
+
+import scala.scalajs.js
+import js.JSConverters._
+import js.annotation._
+
+import parsers._
+
+object ScalaParser {
```

Comment:
intersting idea, do you think the module option of scalajs won't conflict with the existing project?